woocommerce/woocommerce-gutenberg-products-block
# Rename Cart & Checkout totals components

Fixes #2712.

This PR renames some components/folders according to the guidelines in #2712 and with the goal to make component names & folder structure to be aligned with CSS class names.

### Screenshot
![imatge](https://user-images.githubusercontent.com/3616980/101469107-9547c480-3944-11eb-9594-361ec69ec4c4.png)

### How to test the changes in this Pull Request:

If there is any issue, it will probably be apparent when running `npm run build` or `npm run start`, but it's also a good idea to go to the Cart or Checkout block and verify the sidebar looks correct.

In order to get all rows to appear you need to:
* Have a shipping method and in the frontend add a product that needs shipping to the Cart.
* Have taxes enabled and show prices excluding tax (WooCommerce > Settings > Taxes ).
* Have a coupon that adds a discount and introduce it in the frontend.
* Add this code snippet to a PHP file (for example: `woocommerce-gutenberg-products-block.php`) to get the `Fees` entry to appear:
```PHP
add_action( 'woocommerce_cart_calculate_fees', 'add_fee', 10 );
function add_fee( $cart ) {
	$cart->add_fee( __( 'Fee', 'woo-gutenberg-products-block' ), 100, true );
}
```
